cli/daemon/schema.go

```diff
@@ -100,15 +100,15 @@ func (r *schemaRenderer) renderBuiltin(b schema.Builtin) {
 	case schema.Builtin_STRING:
 		r.WriteString("hello")
 	case schema.Builtin_BYTES:
-		r.WriteString("aGV5Cg==") // "hey"
+		r.WriteString("YmFzZTY0Cg==") // "base64"
 	case schema.Builtin_TIME:
 		s, _ := time.Now().MarshalText()
 		r.WriteString(string(s))
 	case schema.Builtin_UUID:
 		r.WriteString("7d42f515-3517-4e76-be13-30880443546f")
 	case schema.Builtin_JSON:
 		r.WriteObjectStart()
-		r.WriteObjectField("jsonData")
+		r.WriteObjectField("some json data")
```

Comment:
These changes where to align with the Go code generator so no matter where we generate the example JSON, it matches